hack/lib/lint.sh

```diff
@@ -18,11 +18,11 @@
 
 set -o errexit
 set -o nounset
-set -o pipefail
 
 kubeedge::lint::check() {
     cd ${KUBEEDGE_ROOT}
     # skip deleted files
+    set +o pipefail
```

Comment:
I recommend that pipefail be enabled by default on line 21  （keep  `set -o pipefail`）

You can cancel pipefail temporarily in this line.  (`set +o pipefail`)

And then finally turn it on at the end of the function. (`set -o pipefail`)

Reply:
Should  turn it on at the end of the function. (set -o pipefail)